### DISCUSSION TITLE
Looking for two tools

### DISCUSSION THREAD

**Discussion by naholt** [community/community#64866] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

good morning, i am looknig for two tools
tool 1: i am looking for a tool that i can run from my pc that will give me a list of all the vlans on a specific switch or switch stack. 
tool 2: i am looking for a tool that will give me all ports not in use on a switch or switch stack. 

i do a lot of network drops and i need to be able to determine if a switch or stack has a certain vlan on it or not without having to have a third party do it.  
also when a network drop in in progress, i have to be able to give the cabling crew a list of available ports to use. 

thank you very much
Nicholas

**Comment by Jaber-Saed** [comment]
Good morning Nicholas,

Thank you for reaching out! It sounds like you're looking for tools that can help you efficiently manage your network infrastructure. Here are some suggestions for the tools you're looking for:

Tool 1: VLAN Discovery Tool

To get a list of all the VLANs on a specific switch or switch stack, you might consider using a network scanning tool or a command-line tool depending on the specific network equipment you're using.

Network Scanning Tools: Tools like SolarWinds Network Performance Monitor, PRTG Network Monitor, or ManageEngine OpManager can help you discover and monitor network devices, including their VLAN configurations.

Command-Line Tools: For Cisco switches, you can use the command-line interface (CLI) to retrieve VLAN information. The command show vlan on Cisco devices can provide you with the VLAN details.

Tool 2: Unused Ports Discovery Tool

To find unused ports on a switch or switch stack, you can again consider using network monitoring tools that can provide this information.

Network Scanning Tools: Tools like SolarWinds Network Performance Monitor and PRTG Network Monitor can not only discover devices but also monitor port status, which includes identifying unused ports.

Switch Management Software: Many network equipment manufacturers offer management software that allows you to view port status, including whether they are in use or not. Examples include Cisco Prime Infrastructure for Cisco devices or HPE Intelligent Management Center for HPE switches.

Remember that the availability and functionality of these tools might depend on the specific network equipment you're using and your network setup. Additionally, some tools might come with licensing costs or limitations based on the number of devices you need to manage.

Before using any of these tools, ensure that they are compatible with your network infrastructure, and consider testing them in a controlled environment to ensure they meet your requirements.

I hope these suggestions help you find the tools you need to manage your network drops more efficiently. If you have any further questions or need more assistance, feel free to ask. Good luck with your network management tasks!

Best regards,
Jaber

**Comment by naholt** [comment]
Thank you very much.  you have given me a path to find what I need.  I am very appreciative.




Regards,
Nicholas A. Holt
Desktop Support 2
Goodman Manufacturing Co., L.P.
A member of Daikin group
19001 Kermier Rd
Waller TX 77484
Phone:  713.346.6378

From: Jaber-Saed ***@***.***>
Sent: Thursday, August 24, 2023 12:38 PM
To: community/community ***@***.***>
Cc: Holt, Nicholas A. ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [community/community] Looking for two tools (Discussion #64866)

[Image removed by sender.]
IRONSCALES couldn't recognize this email as this is the first time you received an email from this sender notifications @ github.com


Good morning Nicholas,

Thank you for reaching out! It sounds like you're looking for tools that can help you efficiently manage your network infrastructure. Here are some suggestions for the tools you're looking for:

Tool 1: VLAN Discovery Tool

To get a list of all the VLANs on a specific switch or switch stack, you might consider using a network scanning tool or a command-line tool depending on the specific network equipment you're using.

Network Scanning Tools: Tools like SolarWinds Network Performance Monitor, PRTG Network Monitor, or ManageEngine OpManager can help you discover and monitor network devices, including their VLAN configurations.

Command-Line Tools: For Cisco switches, you can use the command-line interface (CLI) to retrieve VLAN information. The command show vlan on Cisco devices can provide you with the VLAN details.

Tool 2: Unused Ports Discovery Tool

To find unused ports on a switch or switch stack, you can again consider using network monitoring tools that can provide this information.

Network Scanning Tools: Tools like SolarWinds Network Performance Monitor and PRTG Network Monitor can not only discover devices but also monitor port status, which includes identifying unused ports.

Switch Management Software: Many network equipment manufacturers offer management software that allows you to view port status, including whether they are in use or not. Examples include Cisco Prime Infrastructure for Cisco devices or HPE Intelligent Management Center for HPE switches.

Remember that the availability and functionality of these tools might depend on the specific network equipment you're using and your network setup. Additionally, some tools might come with licensing costs or limitations based on the number of devices you need to manage.

Before using any of these tools, ensure that they are compatible with your network infrastructure, and consider testing them in a controlled environment to ensure they meet your requirements.

I hope these suggestions help you find the tools you need to manage your network drops more efficiently. If you have any further questions or need more assistance, feel free to ask. Good luck with your network management tasks!

Best regards,
Jaber

-
